db/migrate/20161107140328_add_prior_owner_id_to_activities.rb

```diff
@@ -0,0 +1,5 @@
+class AddPriorOwnerIdToActivities < ActiveRecord::Migration[5.0]
+  def change
+    add_column :activities, :prior_owner_id, :integer, index: true
```

Comment:
I kinda like using `:e db/migrate/*owner_id*<tab>` to browse to files in vim... That's all really.